mission_control/test/mission_interface.py

```diff
@@ -76,24 +94,25 @@ def __init__(self):
             '/mission_control_node/execute_mission', ExecuteMission, latch=True, queue_size=1)
 
     def callback_mission_execute_state(self, msg):
-        self.report_execute_mission = msg
+        self.execute_mission_state = msg.execute_mission_state
 
     def callback_mission_load_state(self, msg):
         self.mission_load_state = msg.load_state
 
     def load_mission(self, mission_name):
         # Simulate Jaus Ros Bridge sending a Load Command and wait for
         # response that the mission was loaded correctly
+        self.full_path = os.path.join(
+            os.getcwd(), 'test_files/test_missions/') + mission_name
```

Comment:
@patapufate hmm, why switching from `os.path.abspath(__file__)` to `os.getcwd()`? The former is more robust!

Reply:
Also, consider adding `mission_name` as an `os.path.join` argument instead of using string concatenation.

Reply:
Fixed!